frontend/lib/src/theme/utils.ts

```diff
@@ -287,11 +288,11 @@ export const createEmotionTheme = (
     colors: createEmotionColors(newGenericColors),
     genericFonts: {
       ...genericFonts,
+      ...(headingFont && {
+        headingFont: parseFont(headingFont),
+      }),
       ...(bodyFont && {
-        // We currently do not allow to set different fonts for body and heading
-        // so we use the same font for both.
         bodyFont: parseFont(bodyFont),
-        headingFont: parseFont(bodyFont),
```

Comment:
I think we still want to fallback to use the configured `bodyFont` as `headingFont` in case the `headingFont` is not defined. But maybe check back with @jrieke on this.

Reply:
Yes!